script.module.enum34/lib/enum/enum34.egg-info/top_level.txt

```diff
@@ -0,0 +1 @@
+enum
```

Comment:
remove

Reply:
This file provided by the original author Ethan Furman
Will removing it break the enum package?

Reply:
no this is only needed to packages for pypi

Reply:
OK